src/NRLTB_data.jl

```diff
@@ -22,28 +14,61 @@ function default_orbitals(s)
 end
 
 
-function load_NRLHamiltonian(fname)
-   # HUAJIE TODO
-end
-
+# This function reads the ASCII format data files from the NRL server.
+#
+# The website has been lost, but a functional version remains on the Internet Archive:
+# https://web.archive.org/web/20121003160812/http://cst-www.nrl.navy.mil/bind/
+# 
 function NRLHamiltonian(s; orbitals=default_orbitals(s), cutoff=:forceshift)
```

Comment:
Would it be worthwhile sidelining the "built-in" NRLTB models entirely, e.g. we could make them accessible via a separate constructor, or as a fall-back? I'd like to keep them for testing though.

Reply:
For testing I think having some built in parameters is great. 
Also for tutorials, it allows you to e.g. have a 'metal' (`:Al`) tutorial and then a 'semiconductor' (`:C`) Jupyter notebook, without any further dependencies / downloads. 

Maybe a branching function like:
```
function (NRLHamiltonian)
  if (NRLBUILTIN[:species])
     NRLHamiltonianBuiltIn
  elseif (NRLTABULATED[:species]
     NRLHamiltonainTabulated
   else
    "Can't find species $:species in either built-in or tabulated data."
```

Or could we even dispatch on the species somehow?

Reply:
> Or could we even dispatch on the species somehow?

could work, the dictionary doesn't have to point to a filename